### PR TITLE
Fixed alert history dashboard

### PR DESCRIPTION
**Pull request by gaurav-oc**

### CODE REVIEW

File: kustomize/monitoring/base/monitoring.yaml [hunk 1]
```diff
@@ -777,21 +778,21 @@ data:
...
           "tags": [],
           "templating": {
             "list": [
               {
                 "current": {
-                  "isNone": true,
                   "selected": false,
-                  "text": "None",
-                  "value": ""
+                  "text": "3a6bf0c1-940c-414f-a1f9-5725729b615e",
```
Comment (songole) [inline kustomize/monitoring/base/monitoring.yaml:790]: Is something hardcoded here?

File: kustomize/monitoring/base/monitoring.yaml [hunk 2]
```diff
@@ -636,13 +636,14 @@ data:
           "fiscalYearStartMonth": 0,
           "graphTooltip": 0,
           "id": null,
-          "iteration": 1651813612530,
+          "iteration": 1652333212917,
```
Comment (songole) [inline kustomize/monitoring/base/monitoring.yaml:639]: Something hardcoded here?

Comment (gaurav-oc) [reply kustomize/monitoring/base/monitoring.yaml:639]: No its the number of time Ank might have executed some dashboard

File: kustomize/monitoring/base/monitoring.yaml (hunk 1 above)
Comment (gaurav-oc) [reply kustomize/monitoring/base/monitoring.yaml:790]: No Ank can comment

File: kustomize/monitoring/base/monitoring.yaml (hunk 2 above)
Comment (abikki) [reply kustomize/monitoring/base/monitoring.yaml:639]: @songole This will get changed whenever we make changes to the dashboard, no issues with this.

**Pull request merged**